### ISSUE TITLE
Check polygon for OSBA_1

### ISSUE THREAD

**Issue by slowe** [RobJN/regional-fes#27]
I wanted to flag that the grid supply point `OSBA_1` seems to have two quite widely separated sections to it: one in the North York Moors near Scarborough and the other section is up by the Scottish border. Is this correct or should this be two separate grid supply points?

**Comment by RobJN** [comment]
I agree, this looks strange and is almost certainly wrong. I'll look back through the source data to see if I can work out what's going on here.
